backend/LexBoxApi/Controllers/ProjectController.cs

```diff
@@ -186,4 +186,22 @@ public async Task<ActionResult<HgVerifyResult>> HgVerify(string code)
         var result = await _hgService.VerifyRepo(code);
         return new HgVerifyResult(result);
     }
+
+    [HttpGet("countLexEntries/{code}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesDefaultResponseType]
+    public async Task<ActionResult<int?>> GetLexEntryCount(string code)
+    {
+        return await _projectService.GetLexEntryCount(code);
+    }
+
+    [HttpGet("updateLexEntryCount/{code}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesDefaultResponseType]
+    public async Task<ActionResult<int?>> UpdateLexEntryCount(string code)
+    {
+        return await _projectService.UpdateLexEntryCount(code);
```

Comment:
if null is returned from here then a 204 will be returned. I agree we should return a 404, but we need to be explicit, here and above

Reply:
Good point. I'll catch null and return 404 from it, then change this to be `ActionResult<int>` since at that point if a 200 is returned then it's guaranteed to have a non-null value.